Makefile

```diff
@@ -8,9 +8,6 @@ CONTAINER_ENGINE?=docker
 # Needed for build-catalog-image
```

Comment:
We should be able to get rid of everything in this file except the `include`, the `boilerplate-update` target, and the `generate-syncset` target.

On the latter note, I believe that target was using all those yamls from deploy/. I'm not sure offhand what that's used for -- is it truly replaced by what we're bringing in with boilerplate? If not, those files should be moved to some other directory. I'm pretty sure generate-syncset walks the whole project looking for yamls, so they could be anywhere.

Reply:
> This looks fine mechanically from a boilerplate / appsre pipeline standpoint, but someone CIO should validate the `generate-syncset` thing.

I was going to rip that out. Its not required for the functionality of the cluster and only works in Jenkins since that app-sre python image is private and wont work locally. Its a cool helper script but its not required.